projects/scion/workbench/src/lib/content-projection/content-projection.directive.ts

```diff
@@ -58,9 +58,17 @@ export class ContentProjectionDirective implements OnInit, OnDestroy {
     this._contentViewRef = (await this.overlayHost).createEmbeddedView(this.contentTemplateRef, null);
     this._contentViewRef.detectChanges();
 
-    // (Re-)position content each time the bounding box element changes its size.
+    // Fixed content position.
```

Comment:
// Position projected content out of the document flow relative to the initial page viewport.
`this.styleContent({position: 'fixed'});`

Reply:
done